src/libstd/rt/unwind/seh64_gnu.rs

```diff
@@ -60,8 +60,9 @@ pub struct EXCEPTION_RECORD {
     ExceptionInformation: [ULONG_PTR; 15],
 }
 
-pub type CONTEXT = c_void;
-pub type UNWIND_HISTORY_TABLE = c_void;
+// Opaque structs.
+enum CONTEXT {}
+enum UNWIND_HISTORY_TABLE {}
```

Comment:
I believe these will need to stay `pub` to satisfy the no-private-types-exposed-in-public-signatures check

Other than that can you also remove the `// Opaque struct` comment on both of these? (it's pretty self explanatory).

r=me after those two nits!